.github/workflows/code-quality.yml

```diff
@@ -27,6 +27,9 @@ jobs:
         pip install --pre torchvision -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
         pip install git+https://github.com/pbelevich/transformers.git@compatible_with_pt_master
         pip install "black<23" pylint==v3.0.0a5 mypy==v0.902 flake8 pyre-check==0.9.15
+    - name: Pyre check for SPMD
```

Comment:
Move Pyre to the first place so that we can get signals faster. Mypy runs about 10 mins.